tools/cartopy_feature_download.py

```diff
@@ -114,11 +119,28 @@ def download_features(group_names, dry_run=True):
                         action='store_true')
     parser.add_argument('--ignore-repo-data', action='store_true',
                         help='ignore existing repo data when downloading')
+    parser.add_argument('--no-warn',
+                        action='store_true',
+                        help='ignore cartopy "DownloadWarning" warnings')
     args = parser.parse_args()
 
     if args.output:
-        config['pre_existing_data_dir'] = args.output
-        config['data_dir'] = args.output
+        target_dir = pathlib.Path(args.output).expanduser().resolve()
+        target_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
The `exist_ok` flag was added in Python `3.5`, so I assume it's okay to use here...surely :stuck_out_tongue_winking_eye:

Reply:
I guess you'll just get an error if you want to try with an earlier Python version ;)